src/dynamics.jl

```diff
@@ -14,6 +14,9 @@ k_m = 1.0 #0.1 # [N*m/rpm]
          [-ℓ*k_T 0 ℓ*k_T 0]
          [k_m -k_m k_m -k_m]]
 
+wind = [1,1, 1]*1.0 # wind direction
+wd = 10 # std on wind angle
```

Comment:
Is there any reason in particular you start at 10?

Reply:
Also i think this is the mean instead of the std dev? If that's what the comment indicates

Reply:
Sorry that was a copy pasta from the old quadcopter code.   FIxing